src/fakedht/DhtNode.d

```diff
@@ -91,7 +91,7 @@ public class DhtNode
         params.node_info = this;
 
         Options neo_options;
-        neo_options.cmd_handlers = request_handlers;
+        neo_options.requests = requests;
```

Comment:
Migration notes?

Reply:
Or is this just a deprecation fix?

Reply:
Yeah, this is internal stuff in the node, fixing a deprecation in swarm. Nothing related to the user.

Reply:
Yeah, thanks for the clarification.